src/common/transformations/include/transformations/sdpa_to_paged_attention/position_ids_replacer.hpp

```diff
@@ -23,4 +24,21 @@ class ov::pass::PositionIDsReplacer : public ov::pass::MatcherPass {
 public:
     OPENVINO_RTTI("PositionIDsReplacer", "0");
     explicit PositionIDsReplacer(const Output<Node>& position_ids);
-};
\ No newline at end of file
+};
+
+/**
+ * @brief Qwen model has a specific feature in the model structure not to use position_ids input,
+ * this input is detached. The model expects data processing in order.
```

Comment:
The model uses implicitly defined `position_ids` based on the past KV cache size.

Reply:
done